.mention-bot

```diff
@@ -1,9 +0,0 @@
-{
-  "maxReviewers": 2,
```

Comment:
Did we remove this bot? I've been in and out in of this project and didn't realize it!

@josepjaume did you remove the bot intentionally, or we broke the config unintentionally?